testing/testdata/testmodels2.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto2";
+
+package testmodels;
+
+option go_package = ".;testmodels";
```

Comment:
Eek! Please don't generate Go files where the package is needlessly different than the last element of package path.
```suggestion
option go_package = ".;testdata";
```